### ISSUE TITLE
BAF outputs for X chromosome in males

### ISSUE THREAD

**Issue by asd1289** [hartwigmedical/hmftools#339]
Hi,

I have been using your toolkits for CNA analyses and it seems that the BAF outputs from Amber for male X chr are very sparse. This is evident even in the demo circos plots on Purple GitHub page. I am wondering if this behavior can be changed? Would tweaking the `min_het_af_percent` would be a way to do that? Is there a rationale for this default behavior?

I am using version 3.9 for my analyses.

**Comment by p-priestley** [comment]
We did specifically choose to discard the pseudoautosomal regions from BAF, initially so they wouldn't affect our gender determination and because we felt they didn't have much utility.   Is there a specific reason to add these?

**Comment by asd1289** [comment]
No, I think it is not necessary for my need, therefore closed the issue. Sorry for the inconvenience.
